### PR TITLE
Migrate from oeuftete repo

### PR DESCRIPTION
**Pull request by oeuftete**
Updating the canonical version of this repo from my personal account to the ARe organization.

Once merged, I'll set up the Docker Hub autobuilds analogous to the ones in place for my personal repo.

### CODE REVIEW

**Comment by obaltzer** [comment]
@oeuftete I know it is overkill, but could add a `LICENSE` file with an MIT license here? As it so happens, I am currently going back and forth with legal on our open source contributions.

**Comment by sbchisholm** [comment]
:+1:

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2016 Analyze Re
```
Comment (sbchisholm) [inline LICENSE.txt:1]: :+1:

**Comment by obaltzer** [comment]
:+1:

**Pull request merged**